mozilla/blurts-server
# MNTOR-1632: Port dashboard page

# References: 
Jira: [MNTOR-1632](https://mozilla-hub.atlassian.net/browse/MNTOR-1632)



# Description

Ports the dashboard page to Next.js.

# How to test

- Visit `/user/breaches` and/or `/user/dashboard`
- Visit the dashboard page and make sure that the page looks and works as expected

# Checklist (Definition of Done)
